File: cfgs/java_cfgs_foo/NUMBER_VISIBLE_BOXES_PUTTING_ONE_INSIDE_ANOTHER.java
Nodes
1;special;;;
2;statement;Queue<Integer> q = new LinkedList<>();14;
3;statement;fun_1(foo_1);15;
4;statement;fun_1(foo_2[0]);16;
5;statement;int i = 1;17;
6;conditional;foo_1 < foo_2;18;
7;statement;int now = fun_1();20;
8;conditional;foo_1[foo_2] >= (2 * foo_3);21;
9;statement;fun_1();21;
10;statement;fun_1(foo_2[foo_3]);22;
11;statement;i++;19;
12;statement;return fun_1();24;
13;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;
5;7;12;
6;8;
7;9;10;
8;10;
9;11;
10;6;
11;13;
12;

File: cfgs/pythoncfgs/NUMBER_VISIBLE_BOXES_PUTTING_ONE_INSIDE_ANOTHER.py
Nodes
1;statement;foo_1 = fun_1([]);10;
2;statement;fun_1();11;
3;statement;fun_1(foo_1[0]);12;
4;conditional;foo < foo;13;
5;statement;return fun_1(foo_1);18;
6;statement;foo_1 = foo_2[0];14;
7;conditional;foo_1[foo_2] >= 2 * foo_3;15;
8;statement;fun_1(foo_1[foo_2]);17;
9;statement;fun_1();16;
10;exit;;
Edges
0;2;
1;3;
2;4;
3;5;6;
4;10;
5;7;
6;8;9;
7;4;
8;8;
9;
